electron/generators/diograph-generator.spec.js

```diff
@@ -0,0 +1,164 @@
+const { readPaths } = require('../readers/folder-reader')
+const { generateFileDiory, generateFolderDiory } = require('./diory-generator')
+const { generateDiograph } = require('./diograph-generator')
+
+jest.mock('../readers/folder-reader')
+jest.mock('./diory-generator')
+
+describe('diograph-generator', () => {
```

Comment:
I'd prefer having the function name instead of file name here: `generateDiograph`
```suggestion
describe('generateDiograph', () => {
```

Reply:
ah, I usually have both.

Reply:
Aah, you have also the `generateDiograph` there!